### ISSUE TITLE
Add a config option to make the initial structure spawn in random coordinates

### ISSUE THREAD

**Issue by rafaelkai** [nonamecrackers2/crackers-wither-storm-mod#763]
It would be nice to have this option so we'd have to at least find the structure before being able to summon the Wither Storm. Maybe also being able to configure wether the Wither skulls are present in the structure or not.

**Comment by nonamecrackers2** [comment]
Due to the requirements of having the structure spawn only once, this may not be very feasible. You can change the position of the structure in the mean time however.

In 1.18.2+ you can do this via datapacks. Replace "storm_spawn_platform.json" under "witherstormmod/worldgen/structure" with a new json file containing:

`{
  "type": "witherstormmod:storm_spawn_platform",
  "biomes": "#witherstormmod:has_structure/storm_spawn_platform",
  "adapt_noise": true,
  "spawn_overrides": {},
  "position": [0, 0, 0],
  "step": "surface_structures",
  "terrain_adaptation": "beard_thin"
}`

Replace [0, 0, 0] with any position you like.

**Comment by nonamecrackers2** [comment]
Yeah, currently making the structure spawn at a random position would not be very feasible and you will want to do the above method to change the position (although it wouldn't be random)
